### DISCUSSION TITLE
Custom summary option from the admin page?

### DISCUSSION THREAD

**Discussion by cat-a-flame** [getgrav/grav#3700] [Help]
Hi there!
I've recently started using Garv as my blogging platform, and so far, it meets most of my needs! One thing I couldn't figure out is adding a custom summary to my blog posts on the list page from the admin. Display my posts X characters is not ideal, I want to write custom intros for them. Am I missing something or that is not supported yet?

**Comment by farfallosa** [comment]
Did you find a way to manage it? I face the same challenge.   (GRAV 1.7.49.5 Quark 2.12 Plugin RelatedPage 1.2.5; PHP 8.2.29)

**Reply by cat-a-flame** [reply]
I've abandoned Garv, it wasn't suitable for me, sorry :(

**Reply by farfallosa** [reply]
👍 )
